web/packages/teleport/src/Notifications/Notifications.tsx

```diff
@@ -16,7 +16,7 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-import React, { useState, useEffect, useCallback } from 'react';
+import React, { useState, useMemo, useEffect, useCallback } from 'react';
```

Comment:
FWIW, you can remove the `React` import once it's no longer directly used in a component.

I was going to say that it'd be useful if there was a codemod we could use for this so that we no longer have to remember about it. Turns out there is one and I made an issue about it. 😅 https://github.com/gravitational/teleport/issues/35039

Reply:
oh, ok! for some reason I had it in my head it was a required import for func components even if unused